lib/baseLibrary.js

```diff
@@ -38,9 +40,19 @@ export class BaseLibrary {
 
     const options = merge({}, optionsDefault, opts);
 
+    if (!this.values[value] && options.isOriginalValue && this.compressedValues[value]) {
+       // eslint-disable-next-line no-console
+      console.warn(`WARNING: '${value}' does not exist beforehand in the rename table,` +
```

Comment:
As this code is a duplicate in `selectorLibrary.js` I think it is better to create a static method in BaseLibrary for the logs.

```js
static hasReservedValue(value) {
	// eslint-disable-next-line no-console
	console.warn(`WARNING: '${value}' does not exist beforehand in the rename table,` +
                     ' but appears in the compressed map. Either:');
	// eslint-disable-next-line no-console
	console.warn(`- Create a CSS rule with '${value}' and re-run the process.`);
	// eslint-disable-next-line no-console
	console.warn(`- Add the value to the reserved selectors table so it's not used for renaming.`);

	return `${value}_conflict`;
}
```

And later you could return this `return BaseLibrary.hasReservedValue(value);`. Or maybe you have a better idea :)

Reply:
Yes you're completely right. I'll do this.

Reply:
Done.